Makefile

```diff
@@ -1,19 +1,29 @@
 PREFIX ?= /usr/local
-MAN_PREFIX ?= ${PREFIX}/man
+BINDIR ?= ${PREFIX}/bin
+LIBDIR ?= ${PREFIX}/lib
+SHAREDIR ?= ${PREFIX}/share
+EXAMPLESDIR ?= ${SHAREDIR}/examples
+
+MANDIR.${PREFIX} = ${PREFIX}/share/man
+MANDIR./usr/local = /usr/local/man
+MANDIR. = /usr/share/man
+MANDIR ?= ${MANDIR.${PREFIX}}
```

Comment:
Your GNU foo is better than mine. How does this alter the man directory based on platform? Does it rely on one of the platforms passing `${PREFIX}` at build time?

Reply:
It was a suggestion by someone else -- I wouldn't have thought of this! Basically if you pass `PREFIX=/usr/local` then `MANDIR` is set to `/usr/local/man`, otherwise it's set to `/usr/share/man`. It's very cunning!

Reply:
I bet there's a platform where `PREFIX=/usr/local/`, but the manuals go into `/usr/local/share/man`, but let's not worry about it until we see it.

I thought FreeBSD did it this way, but I just check and I was wrong.